cdap-unit-test/src/test/java/co/cask/cdap/test/app/TestAppWithCube.java

```diff
@@ -51,6 +53,10 @@
  *
  */
 public class TestAppWithCube extends TestBase {
+
+  @ClassRule
+  public static final TestConfiguration CONFIG = new TestConfiguration("explore.enabled", false);
```

Comment:
we should use constants when they exist. I believe in this case a constant exists. (Constants.Explore.EXPLORE_ENABLED)